db/migrate/20150331203445_create_reviews.rb

```diff
@@ -0,0 +1,12 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.integer :rating, null: false
+      t.text    :description, null: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.